src/index.ts

```diff
@@ -23,15 +23,22 @@ export function useFlexSearch <T extends {id: string| number}> (
   watch(
```

Comment:
@terrarium789 Changes in this _watch_ are not reasonable. Please revert changes from this function

Reply:
agree, better with "{" and "}".